### PR TITLE
Ignite add is working!

### PR DESCRIPTION
**Pull request by GantMan**
[28 days ago](https://github.com/infinitered/ignite/pull/498), I had this same thing working, but this time... GLUE GUN!

### Normal add process
image

### When there's a generator collision
image

image

**some caveats:**
* to make my life manageable I only worried about adding generators
* no docs
* code reads like a bad script, needs to become functions
* npm hardcoded over yarn for now

### CODE REVIEW

File: packages/ignite-empty-playground/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   },
   "devDependencies": {
     "ignite": "0.2.0",
-    "ignite-basic-generators": "0.2.0"
+    "ignite-basic-generators": "0.2.0",
+    "ignite-ui": "^16.2.0-RC.2040"
```
Comment (skellock) [inline packages/ignite-empty-playground/package.json]: You'll probably want to remove this.  =)

File: packages/ignite/package.json [hunk 2]
```diff
@@ -27,9 +27,12 @@
   "license": "MIT",
   "dependencies": {
     "gluegun": "0.7.3",
+    "json2toml": "^1.0.6",
```
Comment (skellock) [inline packages/ignite/package.json]: Didn't realize our parser didn't write TOML.

**Comment by skellock** [comment]
I don't think this reads like a bad script at all dude.

File: packages/ignite/package.json (hunk 2 above)
Comment (GantMan) [inline packages/ignite/package.json]: yeah, imagine my shock!  I was loke CMON!!!!!! MANNNNNNN!!!!!

File: packages/ignite-empty-playground/package.json (hunk 1 above)
Comment (GantMan) [inline packages/ignite-empty-playground/package.json]: bahahaha, I searched npm for something that started with `ignite-`

**Pull request merged**